cmd/diff-main.go

```diff
@@ -35,7 +35,7 @@ var (
 // Compute differences between two files or folders.
 var diffCmd = cli.Command{
 	Name:   "diff",
-	Usage:  "List objects with size difference or missing between two folders or buckets.",
+	Usage:  "List differences in object name, size, and date between folders.",
```

Comment:
I think 'two folders' will still be more clear.

Reply:
@vadmeste,
`between` in English is inherently involves 2 objects/things, unlike `among`, which is used when many or more than 2 items/units are involved in the action. That's why, there is no need to say here `two folders` explicitly.

Reply:
> @vadmeste,
> between in English is inherently involves 2 objects/things, unlike among, which is used when many or more than 2 items/units are involved in the action. That's why, there is no need to say here two folders explicitly.

:+1: